TrayTip/__init__.py

```diff
@@ -62,7 +62,7 @@ def OnNotify(self, hwnd, msg, wparam, lparam):
         if lparam == 0x0404 or lparam == 0x0405:
             win32gui.DestroyWindow(hwnd)
 
-class showTip(eg.ActionBase):
+class ShowTip(eg.ActionBase):
```

Comment:
yeah you would. My bad. but also the Text class as well because you merged those changes.

But Make sure you test any of the PR's i submit. I have been known to fat finger the hell out of things. or overlook things like this.